app/config/webpack_encore.yaml

```diff
@@ -0,0 +1,15 @@
+webpack_encore:
+  # The path where Encore is building the assets.
+  # This should match Encore.setOutputPath() in webpack.config.js.
+  output_path: '%kernel.project_dir%/web/build'
+  # If multiple builds are defined (as shown below), you can disable the default build:
+  # output_path: false
+
+  # if using Encore.enableIntegrityHashes() specify the crossorigin attribute value (default: false, or use 'anonymous' or 'use-credentials')
+  # crossorigin: 'anonymous'
+
+  # Cache the entrypoints.json (rebuild Symfony's cache when entrypoints.json changes).
+  # To enable caching for the production environment, creating a webpack_encore.yaml in the config/packages/prod directory with this value set to true
+  # Available in version 1.2
+  #cache: false
```

Comment:
On Stepup-RA (prod specific config), the cache is enabled for webpack_encore. Consider doing that here to by adding a config/packates/prod/webpack_encore.yaml and override the cache setting there.